assets/stylesheets/units/_headings.scss

```diff
@@ -2,7 +2,8 @@ h1 {
   @include bold-font(64);
 }
 
-h2 {
+h2,
+.heading__level-2 {
```

Comment:
Makes sense to me, will write something along the lines of a modifier instead of coupling to header sizes.